webservices/src/main/java/com/javaschool/webservices/model/ShippingRabbitRPCMessage.java

```diff
@@ -0,0 +1,18 @@
+package com.javaschool.webservices.model;
+
+public class ShippingRabbitRPCMessage {
```

Comment:
If you are only using this class for the enum above, you can make it package private.

Reply:
I made it package private.